src/SDKs/ContainerRegistry/Management.ContainerRegistry/Microsoft.Azure.Management.ContainerRegistry.csproj

```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.ContainerRegistry</PackageId>
 		<Description>Provides management capabilities for Azure Container Registry.</Description>
-		<Version>2.0.0</Version>
+		<Version>2.1.0-preview</Version>
 		<AssemblyName>Microsoft.Azure.Management.ContainerRegistry</AssemblyName>
 		<PackageTags>Microsoft Azure Container Registry;Container Registry;</PackageTags>
 		<PackageReleaseNotes>
```

Comment:
@northtyphoon please add packageReleaseNotes for this release. This is viewable on nuget.org so that you customers know what do they get in this release.

Reply:
Update

---
In reply to: [207694565](https://github.com/Azure/azure-sdk-for-net/pull/4630#discussion_r207694565) [](ancestors = 207694565)